### ISSUE TITLE
Add more complete trace example

### ISSUE THREAD

**Issue by kintel** [GoogleCloudPlatform/opentelemetry-operations-js#25]
The current trace example is very minimal.
We should add a more complete example that:
* Is a service
* Makes an external HTTP request
* Uses auto-instrumentation to create both server and client spans
* Uses trace context from incoming HTTP headers

**Comment by mayurkale22** [comment]
Just so you know: we have a couple of examples in the main OpenTelemetry repo: https://github.com/open-telemetry/opentelemetry-js/tree/master/examples. Maybe, we can point everyone to use those examples, WDYT?

**Comment by kintel** [comment]
Yeah, I don't think we need to be exhaustive here, but since none of the upstream examples will work out of the box with Cloud Trace, it would be nice with at least one more complete example here.
The goal would be to provide something that works out of the box to help customers get started.

These examples _could_ live in https://github.com/GoogleCloudPlatform/nodejs-docs-samples, but I'm not 100% certain what examples should go where.

**Comment by dashpole** [comment]
triaging.  It is a good suggestion, but not urgent.

**Comment by dashpole** [comment]
Still not urgent
